scene/resources/fog_material.cpp

```diff
@@ -159,7 +159,7 @@ uniform sampler3D density_texture: hint_default_white;
 void fog() {
     DENSITY = density * clamp(exp2(-height_falloff * (WORLD_POSITION.y - OBJECT_POSITION.y)), 0.0, 1.0);
     DENSITY *= texture(density_texture, UVW).r;
-    DENSITY *= pow(clamp(-SDF / min(min(EXTENTS.x, EXTENTS.y), EXTENTS.z), 0.0, 1.0), edge_fade);
+    DENSITY *= pow(clamp(-2 * SDF / min(min(SIZE.x, SIZE.y), SIZE.z), 0.0, 1.0), edge_fade);
```

Comment:
```suggestion
    DENSITY *= pow(clamp(-2.0 * SDF / min(min(SIZE.x, SIZE.y), SIZE.z), 0.0, 1.0), edge_fade);
```
Needs to be a float or the shader will fail to compile

Reply:
> Do you have time in the next 14 hours or so to add this? If not I will happily do it

Yes, will working on it right now. :)

Reply:
Damn, thanks!